src/pages/verify/e-ids/swedish-bankid.mdx

```diff
@@ -206,6 +206,107 @@ HTTP/1.1 400 Bad Request
 }
 ```
 
+## BankID Headless Authentication
+
+### Overview
+
+Headless authentication is designed for flows where you want maximum control over how user interacts with same device and QR flows for BankID.
+
+For instance you could render the QR flow in your own website, or trigger app2app in your native app without having to open a browser
+
+### The underlying Headless flow
+
+Criipto Verify facilitates Swedish BankID headless authentication via the [Headless flow](/verify/getting-started/oidc-intro/#authenticate-headless).
+
+### Implementing BankID Headless Authentication
+
+The Headless flow is only available to traditional server-based web applications (confidential clients) and requires client validation for authentication requests. We'll use [private key JWTs](/verify/guides/privatekey-jwt/) for authenticaiton in the example below. An alternative (*less secure*) approach is to use `client_secret` for authentication.
+
+1. `HTTP POST` request to the `/headless/authorize` endpoint:
+
+```sh
+# Replace `$client_assertion` with your JWT assertion
+# Replace `YOUR_DOMAIN.criipto.id` with your Criipto domain
+
+HTTP POST https://YOUR_DOMAIN.criipto.id/headless/authorize
+Content-Type: application/x-www-form-urlencoded
+
+scope=openid
+&end_user_ip=127.0.0.1
+&acr_values=urn:grn:authn:se:bankid
+&client_assertion_type=urn:ietf:params:oauth:client-assertion-type:jwt-bearer
+&client_assertion=$client_assertion
+```
+
+2. A successful response will look something like this:
+
+```sh
+{ 
+  "headless_req_id" : "3857f8ff-21b9-48ae-a732-a3bd8128a7ae",
+  "expires_in" : 120,
+  "acr_data": {
+    "qrData": "bankid.5367b735-a3f9-47d9-aafc-6d2ad5a14901.0.425c66c472e756fc6a070b86eef1adb4fac0a2714d828c902afa77a5b15e8590",
+    "autoStartToken": "d2c96ea2-f8d9-4e9f-8d39-4f2e88e10e1c"
+  }
+}
+```
+
+3. Poll the token endpoint (`/oauth2/token`) to get a response with an ID token.  
+   The `headless_req_id` is used to construct a poll request on the token endpoint:
+
+```sh
+# Replace `HEADLESS_REQ_ID` with the id returned by authentication response
+# Replace `$client_assertion` with your JWT assertion
+
+HTTP POST https://YOUR_DOMAIN.criipto.id/oauth2/token
+Content-Type: application/x-www-form-urlencoded
+
+headless_req_id=HEADLESS_REQ_ID
+&grant_type=urn:urn:params:grant-type:headless
+&client_assertion_type=urn:ietf:params:oauth:client-assertion-type:jwt-bearer
+&client_assertion=$client_assertion
+```
+
+_**Polling Frequency:** Bear in mind, that the authentication process requires interacting multiple times with the end-user, so the identification process could take some time. Consequently, the token endpoint should be polled every 5 seconds._
```

Comment:
Maybe this could be in one of the warning boxes instead of being bolded/italicized?